site/oauth2-examples-azure.md

```diff
@@ -118,6 +118,13 @@ Now that some roles have been created for your application, you still need to as
 
 9. Repeat the operations for all the roles you want to assign.
 
+## About custom signing keys
+
+You do not need to create a custom signing key for your application. If you create one though, you must append an `appid` query parameter containing the *app ID* to the  `jwks_uri`.
```

Comment:
How about changing the above to: "It is optional to create a signing key for your application. If you create one, you must......."